runtime/Cpp/runtime/src/TokenStreamRewriter.cpp

```diff
@@ -305,7 +305,9 @@ std::string TokenStreamRewriter::getText(const std::string &programName, const I
 }
 
 std::unordered_map<size_t, TokenStreamRewriter::RewriteOperation*> TokenStreamRewriter::reduceToSingleOperationPerIndex(
-  std::vector<TokenStreamRewriter::RewriteOperation*> rewrites) {
+  std::vector<TokenStreamRewriter::RewriteOperation*> *rewrites_ptr) {
+
+  std::vector<TokenStreamRewriter::RewriteOperation*> &rewrites = *rewrites_ptr;
```

Comment:
This is not needed. Just restore the original parameter list, but make `rewrites` a reference as well and that's it.